### DISCUSSION TITLE
桌面版隧道分流

### DISCUSSION THREAD

**Discussion by Hazel9067** [getlantern/lantern-forum-cn#270] [Q&A]
桌面版可以像安卓系统一样进行隧道分流吗？
好像只有勾选管理系统代理和代理全部流量，不能设置是否代理某些特定的网站。
可以完善这一功能吗？
<img width="431" alt="Screenshot 2024-03-24 211548" src="https://github.com/getlantern/lantern-forum-cn/assets/164753642/15b5ea4d-a817-4ec4-9cdb-18014e82a3d7">

**Comment by MaxMKMax** [comment]
@Hazel9067 您好，谢谢您的反馈。好的, 我会帮您和科技部沟通您需要的需求!

您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。

1. go to Support (转到支持)
2. select Report an Issue (选择报告问题)
3. choose issue type (选择问题类型)
4. include additional details (包括额外的详细信息)
5. hit Send Report (点击发送报告)

当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队.

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。
我们将非常感谢您。
